### PR TITLE
Add a chart with hardcoded data

### PR DESCRIPTION
**Pull request by shaargtz**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -38,6 +42,28 @@ function createMap() {
...
+ * Draws a chart and adds it to the page.
+ */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('date', 'Year');
+  data.addColumn('number', 'Interest');
+        data.addRow([new Date(2020, 1), 85]);
+        data.addRow([new Date(2020, 2), 85]);
+        data.addRow([new Date(2020, 3), 97]);
+        data.addRow([new Date(2020, 4), 100]);
```
Comment (timothy-parsons) [inline portfolio/src/main/webapp/script.js:55]: The indentation for these lines looks pretty off.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -38,6 +42,28 @@ function createMap() {
       });
 }
 
+/**
+ * Draws a chart and adds it to the page.
```
Comment (timothy-parsons) [inline portfolio/src/main/webapp/script.js:46]: Can you clarify that it is hardcoded data at the moment? And if you are planning on changing that in the future, can you add a TODO?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (shaargtz) [reply portfolio/src/main/webapp/script.js]: Done!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (shaargtz) [reply portfolio/src/main/webapp/script.js]: Done!

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -38,6 +42,29 @@ function createMap() {
       });
 }
 
+/**
+ * Draws a chart and adds it to the page. Currently uses hardcoded data.
+ * TODO: get data from server.
```
Comment (timothy-parsons) [inline portfolio/src/main/webapp/script.js:47]: nit: Can you make your TODOs in the format 'TODO(shaargtz)'? That way if you're working on a codebase with multiple people (like during your capstone!) it'll be clear who is supposed to be doing what.

Comment (shaargtz) [reply portfolio/src/main/webapp/script.js]: Oh, alright, didn't know that was the way to handle it. Done!

**Pull request merged**